charts/newrelic-infrastructure-v3/templates/NOTES.txt

```diff
@@ -13,6 +13,149 @@ Those components are needed to have the full experience on NROne kubernetes expl
 
 {{- end }}
 
+{{- if .Values.nodeSelector }}
+
+###########
+# WARNING #
+###########
+
+You have specified "nodeSelector" into values, however the new version of the chart
+support different nodeSelectors per each component, therefore the value specified has been ignored.
+
+Please use ksm.nodeSelector, controlPlane.nodeSelector and kubelet.nodeSelector.
+
+{{- end }}
```

Comment:
A cheap alternative could be to emit this warning but render the legacy `nodeSelector` for all three components. Do you see any potential drawback for this?

Reply:
no drawback in this case, I've added that